scip.proto

```diff
@@ -369,3 +375,88 @@ enum DiagnosticTag {
   Unnecessary = 1;
   Deprecated = 2;
 }
+
+// Language standardises names of common programming languages that can be used
+// for the `Document.language` field. The primary purpose of this enum is to
+// prevent a situation where we have a single programming language ends up with
+// multiple string representations. For example, the C++ language uses the name
+// "C_PlusPlus" in this enum and other names such as "cpp" are incompatible.
+// Feel free to send a pull-request to add missing programming languages.
+enum Language {
+  UnspecifiedLanguage = 0;
```

Comment:
Is there a specific reason these are kinda in haphazard order? It would be nice to have related languages be grouped to some extent.